testing/sliding-sync-integration-test/src/lib.rs

```diff
@@ -107,8 +335,8 @@ mod tests {
         let build_view = |name| {
             SlidingSyncViewBuilder::default()
                 .sync_mode(SlidingSyncMode::Selective)
-                .set_range(0u32, 10u32)
-                .sort(vec!["by_recency".to_owned(), "by_name".to_owned()])
+                .add_range(0u32, 10u32)
+                .sort(vec!["by_recency".to_string(), "by_name".to_string()])
```

Comment:
You seem to have reverted a change from `to_string` to `to_owned` here (and below).

Reply:
:+1: